src/environments/auth/keycloak-config.dev.ts

```diff
@@ -31,7 +31,7 @@ const keycloakAuthenticationProviders: AuthProviders = {
 };
 
 const keycloakConfigDev: KeycloakConfig = {
-  url: 'http://localhost:8082/auth/',
+  url: 'http://localhost:8081/auth/',
```

Comment:
this change should only be in 11.2 onwards. So remove it here and add it when you merge these changes into 11.2